components/zoom_admin/actions/delete-cloud-recording/delete-cloud-recording.mjs

```diff
@@ -0,0 +1,45 @@
+import zoomAdmin from "../../zoom_admin.app.mjs";
+import isObject from "lodash/isObject.js";
+import { axios } from "@pipedream/platform";
+import consts from "../../consts.mjs";
+
+export default {
+  name: "Delete Cloud Recording",
+  description: "Remove a recording from a meeting or webinar. [See the docs here](https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/recordingdeleteone)",
+  key: "zoom_admin-action-delete-cloud-recording",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    zoomAdmin,
+    cloudRecording: {
```

Comment:
@vellames-turing I also get duplicated items listed in the async options dropdown here

Reply:
Hey @dannyroosevelt, looks normal to me, when you save some record, dropbox creates two registers, MP4 and M4A, as far as I know, M4A is just for audio.
![Screen Shot 2022-01-20 at 2 19 05 PM](https://user-images.githubusercontent.com/91911778/150389216-6b47c972-6b53-47bf-96ac-0279f2b41612.png)

Reply:
Ah good call, thanks. So actually shouldn't I see more recordings in this list? 
![image](https://user-images.githubusercontent.com/20780753/150391644-4e06559b-c58e-41de-ae8f-5b0cb5abc639.png)
![image](https://user-images.githubusercontent.com/20780753/150391591-51e1e3d2-663a-4620-8bf7-107d94ccc20d.png)

Reply:
Indeed, I will take a look